brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaNodeToEntityConverter.java

```diff
@@ -200,17 +200,18 @@ public ToscaNodeToEntityConverter setTreeBuilder(TopologyTreeBuilderService tree
             spec.configure(VanillaSoftwareProcess.LAUNCH_COMMAND, "true");
             spec.configure(VanillaSoftwareProcess.STOP_COMMAND, "true");
             spec.configure(VanillaSoftwareProcess.CHECK_RUNNING_COMMAND, "true");
-        }
 
-        Optional<PaaSNodeTemplate> optionalPaaSNodeTemplate = getPaasNodeTemplate();
-        if(optionalPaaSNodeTemplate.isPresent()) {
-            Map<String, PaaSNodeTemplate> allNodes = treeBuilder.buildPaaSTopology(topology).getAllNodes();
-            for (Map.Entry<String, IValue> attribute : indexedNodeTemplate.getAttributes().entrySet()) {
-                String value = FunctionEvaluator.parseAttribute(attribute.getKey(), attribute.getValue(), topology, ImmutableMap.<String, Map<String, InstanceInformation>>of(), "", optionalPaaSNodeTemplate.get(), allNodes);
-                spec.addInitializer(new StaticSensor<String>(ConfigBag.newInstance()
-                        .configure(StaticSensor.SENSOR_NAME, "tosca.attribute." + attribute.getKey().replaceAll("\\s+", "."))
-                        .configure(StaticSensor.STATIC_VALUE, value)
-                ));
+            // add static sensors for defined attributes
```

Comment:
Because this is exactly what was causing the problem - if it is not a VanillaSoftwareProcess, then entities will already have been given attributes from the brooklyn plugin, and there is no way, as far as I can tell, to make sure that sensors don't already exist.  If we could formulate the logic only add sensor here if attribute doesn't come from the plugin making an attribute from sensors we'd be fine.